src/python/WMCore/WMBS/MySQL/Create.py

```diff
@@ -65,6 +65,26 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE UNIQUE INDEX uniq_wmbs_file_run_lumi on
              wmbs_file_runlumi_map (fileid, run, lumi)"""
 
+        #
+        # Insert LogCollect site to account for LogCollect output
+        #
+        self.inserts["99_location_logcollect_1"] = \
```

Comment:
Haha, what we have now is way more hacky (registering a file that is present at T0_CH_CERN_MSS with the T2_CH_CERN PNN).

This is correct. We stage out to T0_CH_CERN_MSS, we should register the files as present at T0_CH_CERN_MSS and the only way to do that is to have a site that has that as PNN. No real processing site has it, so I needed to add one.